a3fe/tests/test_engine_configuration.py

```diff
@@ -198,12 +198,12 @@ def test_compare_with_reference_config():
             assert line in cfg_content, f"Expected '{line}' in generated config."
 
 
-def test_copy_from_existing_config():
+def test_copy_from_existing_config(engine_config):
     """Test that we can copy from an existing somd.cfg file."""
     reference_config = "/home/roy/software/deve/a3fe/a3fe/data/example_run_dir/bound/discharge/output/lambda_0.000/run_01/somd.cfg"
```

Comment:
Please avoid hard-coded path so that it works on any workstation. Thanks! You can always try running tests on plato to make sure things work with different slurm settings and paths.